features/search.feature

```diff
@@ -4,13 +4,13 @@ Feature: Search
   I need to be able to search for a word
 
   Scenario: Searching for a page that does exist
-    Given I am on "/wiki/Main_Page"
+    Given I am on "http://en.wikipedia.org/wiki/Main_Page"
```

Comment:
Why you change this? It already set in configuration https://github.com/Behat/MinkExtension/pull/282/files#diff-7bde54db60a6e933518d8b61b929edceR10